symphony-bdk-core/src/test/java/com/symphony/bdk/core/OboServicesTest.java

```diff
@@ -0,0 +1,36 @@
+package com.symphony.bdk.core;
+
+import static org.mockito.Mockito.mock;
+
+import com.symphony.bdk.core.auth.AuthSession;
+import com.symphony.bdk.core.config.model.BdkConfig;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class OboServicesTest {
+
+  private OboServices oboServices;
+
+  @BeforeEach
+  void setUp() {
+    oboServices = new OboServices(new BdkConfig(), mock(AuthSession.class));
+    Assertions.assertNotNull(oboServices.messages());
```

Comment:
static import of `assertNotNull`

Reply:
`assertNotNull(oboServices.messages())` is already tested by `testOboMessages()`